app/controllers/publications_controller.rb

```diff
@@ -9,6 +9,11 @@ def index
     @filter.publications_search
 
     respond_to do |format|
+      @content_item = Whitehall
```

Comment:
It wouldn't be incorrect to be outside, but since this is only used in ERB partials to add meta tags to the `<head>` section of the page, we don't need to fetch it for the other media types.

By adding it for all media types, I would also need to change a lot more tests to stub the content store call, so I decided to keep it isolated.

Does that make sense?